### DISCUSSION TITLE
Is there any way to white-list/black-list some paths/urls?

### DISCUSSION THREAD

**Discussion by kcsujeet** [arthurfiorette/axios-cache-interceptor#753] [Q&A] [answered]
I was wondering if I could white-list some paths so that data is not cached for them. Similar to the `exclude -> path`  option in `axios-cache-adapter` https://github.com/lisaogren/axios-cache-adapter?tab=readme-ov-file#options

**Comment by arthurfiorette** [comment]
Hey @kcsujeet, surprisingly not, you must use [`cache: false`](https://axios-cache-interceptor.js.org/config/request-specifics#cache) on each on of them. However, this is an awesome feature to be implemented, you're up to a PR?

**Reply by kcsujeet** [reply, marked as answer]
Hey @arthurfiorette , I've created a PR [here](https://github.com/arthurfiorette/axios-cache-interceptor/pull/754). It's working but not sure if it's the correct approach. I spun this up in like 2 hours. Let me know if you want any changes there.

**Comment by thomassth** [comment]
With release v1.8.0, whitelisting is now also possible

https://github.com/arthurfiorette/axios-cache-interceptor/pull/1008
